src/App.js

```diff
@@ -1,25 +1,50 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import Dropdown from './dropdown.jsx';
+import List from './pokeList.jsx';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+class App extends Component {
+  state = { pokedex: [], query: null };
```

Comment:
Your default data being saved into State should always reflect the data TYPE that you will actually be using. In this example, query should be an empty string since the data type being stored will be a string.